raiden/transfer/channel.py

```diff
@@ -1431,20 +1431,22 @@ def events_for_close(
     return events
 
 
-def events_for_expired_lock(
-        channel_state: NettingChannelState,
-        secrethash: typing.SecretHash,
+def create_sendexpiredlock(
+        sender_end_state: NettingChannelEndState,
         locked_lock: HashTimeLockState,
         pseudo_random_generator: random.Random,
-) -> typing.List[SendLockExpired]:
-    nonce = get_next_nonce(channel_state.our_state)
-    locked_amount = get_amount_locked(channel_state.our_state)
-    our_balance_proof = channel_state.our_state.balance_proof
+        chain_id: typing.ChainID,
+        token_network_identifier: typing.TokenNetworkID,
+        channel_identifier: typing.ChannelID,
+        recipient: typing.Address,
+):
```

Comment:
A return type annotation would be nice

Reply:
It was added in: https://github.com/raiden-network/raiden/pull/2846/commits/d23666acf8ade6b67f3c4455e43b1546efe1d08b